### ISSUE TITLE
Agents can't actually run agent-specific code!

### ISSUE THREAD

**Issue by qiemem** [arthurhjorth/LevelSpaceXWGUI#30]
To reproduce, make this command:

![image](https://cloud.githubusercontent.com/assets/28215/7439115/93c46fe4-f036-11e4-9f8a-9adb4686112b.png)

Put it in this relationship:

![image](https://cloud.githubusercontent.com/assets/28215/7439121/9d51ad4c-f036-11e4-8a74-59273dcd1b50.png)

and hit Run. You'll get:

```
this code can't be run by the observer, only a turtle
error while observer running FD
  called by (command task from: procedure __EVALUATOR)
  called by procedure __EVALUATOR
```

**Comment by arthurhjorth** [comment]
That's bad. And odd, because I know I tested that exact case. I wonder what has changed... I'll see if I can find out.

**Comment by qiemem** [comment]
Fixed by #44
